README.md

```diff
@@ -1,170 +1,182 @@
-    - Ensures the stability of the network interface name
-
-  * Kernel
-    - Configure virtual NUMA (disable for kernel <2.6.37)
-    - Consume Hyper-V entropy for /dev/random
-    - Configure SCSI timeouts for the root device (which could be remote)
-
-  * Diagnostics
-    - Console redirection to the serial port
-
-  * SCVMM Deployments
-    - Detect and bootstrap the VMM agent for Linux when running in a System
-      Center Virtual Machine Manager 2012R2 environment
-
-  * VM Extension
-    - Inject component authored by Microsoft and Partners into Linux VM (IaaS)
-      to enable software and configuration automation
-    - VM Extension reference implementation on https://github.com/Azure/azure-linux-extensions
-
-
-### COMMUNICATION
+# Microsoft Azure Linux Agent
+
+## Build
+
+Image|Status
+---|---|
+Canonical UbuntuServer 14.04.5-LTS|![badge](https://dcrbadges.blob.core.windows.net/scenarios/Canonical_UbuntuServer_14.04.5-LTS__agent--bvt.svg)
+Canonical UbuntuServer 14.04.5-DAILY-LTS|![badge](https://dcrbadges.blob.core.windows.net/scenarios/Canonical_UbuntuServer_14.04.5-DAILY-LTS__agent--bvt.svg)
+Canonical UbuntuServer 16.04-LTS|![badge](https://dcrbadges.blob.core.windows.net/scenarios/Canonical_UbuntuServer_16.04-LTS__agent--bvt.svg)
+Canonical UbuntuServer 16.04-DAILY-LTS|![badge](https://dcrbadges.blob.core.windows.net/scenarios/Canonical_UbuntuServer_16.04-DAILY-LTS__agent--bvt.svg)
+Canonical UbuntuServer 18.04-LTS|![badge](https://dcrbadges.blob.core.windows.net/scenarios/Canonical_UbuntuServer_18.04-LTS__agent--bvt.svg)
+Canonical UbuntuServer 18.04-DAILY-LTS|![badge](https://dcrbadges.blob.core.windows.net/scenarios/Canonical_UbuntuServer_18.04-DAILY-LTS__agent--bvt.svg)
+Credativ Debian 8|![badge](https://dcrbadges.blob.core.windows.net/scenarios/Credativ_Debian_8__agent--bvt.svg)
+Credativ Debian 8-DAILY|![badge](https://dcrbadges.blob.core.windows.net/scenarios/Credativ_Debian_8-DAILY__agent--bvt.svg)
+Credativ Debian 9|![badge](https://dcrbadges.blob.core.windows.net/scenarios/Credativ_Debian_9__agent--bvt.svg)
+Credativ Debian 9-DAILY|![badge](https://dcrbadges.blob.core.windows.net/scenarios/Credativ_Debian_9-DAILY__agent--bvt.svg)
+OpenLogic CentOS 6.9|![badge](https://dcrbadges.blob.core.windows.net/scenarios/OpenLogic_CentOS_6.9__agent--bvt.svg)
+OpenLogic CentOS 7.4|![badge](https://dcrbadges.blob.core.windows.net/scenarios/OpenLogic_CentOS_7.4__agent--bvt.svg)
+RedHat RHEL 6.9|![badge](https://dcrbadges.blob.core.windows.net/scenarios/RedHat_RHEL_6.9__agent--bvt.svg)
+RedHat RHEL 7-RAW|![badge](https://dcrbadges.blob.core.windows.net/scenarios/RedHat_RHEL_7-RAW__agent--bvt.svg)
+SUSE SLES 12-SP2|![badge](https://dcrbadges.blob.core.windows.net/scenarios/SUSE_SLES_12-SP2__agent--bvt.svg)
+
+## Introduction
+
+The Microsoft Azure Linux Agent (waagent) manages Linux and FreeBSD provisioning and VM interaction with the Azure Fabric Controller. It provides the following
```

Comment:
This should be Linux only. We don't validate FreeBSD even though there is code to support it and few other different UNIX flavors.

Reply:
Updated